lib/OperatingSystems/IOperatingSystem.php

```diff
@@ -17,34 +18,15 @@
 interface IOperatingSystem {
 	public function supported(): bool;
 
-	/**
-	 * Get name of the processor.
-	 *
-	 * @return string
-	 */
-	public function getCpuName(): string;
+	public function getCPU(): CPU;
 
-	/**
-	 * Get number of processors (threads).
-	 *
-	 * @return int
-	 */
-	public function getCpuCount(): int;
+	public function getMemory(): Memory;
 
 	/**
-	 * Get disk info returns a list of Disk objects. Used and Available in bytes.
-	 *
 	 * @return Disk[]
 	 */
 	public function getDiskInfo(): array;
 
-	/**
-	 * Get memory returns a Memory object. All values are in bytes.
```

Comment:
Why do you remove the descriptions? They seem useful.

Reply:
Because it's outdated, the values are in MB. 

https://github.com/nextcloud/serverinfo/blob/326cc462130be484304ad4ab463567df2d87f45b/lib/Resources/Disk.php#L39
https://github.com/nextcloud/serverinfo/blob/326cc462130be484304ad4ab463567df2d87f45b/lib/Resources/Memory.php#L29